library/attachment-viewer/src/main/res/layout/activity_attachment_viewer.xml

```diff
@@ -1,45 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/rootContainer"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".AttachmentViewerActivity">
+        xmlns:tools="http://schemas.android.com/tools"
```

Comment:
I do not understand why we now have 8 spaces for an indent.

Reply:
A mistake. Good spot!